app/main.py

```diff
@@ -1,8 +1,51 @@
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self, comfort_class: int,
+            clean_mark: int, brand: str) -> None:
+
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self, distance_from_city_center: int,
+            clean_power: int, average_rating: int,
+            count_of_ratings: int) -> None:
+
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        income = 0
+        for car in cars:
+            if car.clean_mark <= self.clean_power:
+                income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return income
+
+    def calculate_washing_price(self, car: Car) -> float:
+        diff_clean_power = self.clean_power - car.clean_mark
+        wash_price_wo_city_center = (car.comfort_class
+                                     * diff_clean_power
+                                     * self.average_rating)
+        washing_price_with_cc = (wash_price_wo_city_center
+                                 / self.distance_from_city_center)
```

Comment:
Your line break look not so good. Can you find out way to get rid of them?

Reply:
I made some additional method to solve this.Is it correct form to fix this issuer?